Sources/StreamChat/StateLayer/ChannelList.swift

```diff
@@ -68,13 +68,11 @@ public class ChannelList {
     ///
     /// - Throws: An error while communicating with the Stream API.
     /// - Returns: An array of loaded channels.
-    @discardableResult public func loadMoreChannels(limit: Int? = nil) async throws -> [ChatChannel] {
-        let limit = limit ?? query.pagination.pageSize
-        let count = await state.channels.count
-        return try await channelListUpdater.loadNextChannels(
+    @discardableResult public func loadMoreChannels(limit: Int = 20) async throws -> [ChatChannel] {
```

Comment:
Btw, I know I added this comment previously, but actually, the `limit: Int?` being optional has a valid reason. 

If the limit is `nil` which should be the default, then we should use the pagination size of the `self.query`.

This is so that, whenever the customer initialises the `ChannelList` with a given query, and sets the pagination size there, it should use by default the pagination size of the initial query. So all limits, should be like so:
`let limit = limit ?? self.query.pagination.pageSize ?? defaultValue`